scintilla/win32/DepGen.py

```diff
@@ -13,8 +13,8 @@
 topComment = "# Created by DepGen.py. To recreate, run DepGen.py.\n"
 
 def Generate():
-	sources = ["../src/*.cxx", "../lexlib/*.cxx", "../lexers/*.cxx"]
-	includes = ["../include", "../src", "../lexlib"]
+	sources = ["../src/*.cxx", "../lexlib/*.cxx", "../lexers/*.cxx", "../boostregex/*.cxx"]
+	includes = ["../include", "../src", "../lexlib", "../boostregex"]
```

Comment:
This part is Python script used for Scintilla project, so I think it's not necessary to modify, since this script won't be used for the generation of libScilexer. Or am I missing somethings?